on-chain/shell.nix

```diff
@@ -8,7 +8,6 @@ in
 pkgs.mkShellNoCC {
   packages = with pkgs; [
     gnumake
-    nodejs_18
-    jq
+    nodejs_20
```

Comment:
Not a big deal, but any reason for the update?  It seemed okay on 18.  Did you add a v20 feature?

Reply:
No, just what I had on my machine and I wanted to check if it removed some deprecation messages. Either way, but I figured using later is better. We can roll back if we need to.